packages/blade/src/components/DatePicker/DateInput.web.tsx

```diff
@@ -92,12 +92,13 @@ const _DatePickerInput = (
     successText,
     errorText,
     helpText,
+    dateFormat,
+    placeholder,
```

Comment:
Its fine to not expose placeholder right? dateFormat can have its own defined placeholder I'm guessing 🤔

Reply:
I think we should expose the placeholder.
If someone wants to change it from "DD/MM/YYYY" to something like "Select Date," it should be allowed.
developers may want to modify the placeholder to suit their needs.
We should allow developers to modify the placeholder because having a predefined one could impose unnecessary constraints.

Reply:
Yeah fine. just keep the default as it was before.

Reply:
If we set placeholder to "Select Date" how would the end-user know which format they are supposed to enter date in 🤔 

Works to have placeholder for now since we'll have good default anyways. Can discuss it later.

Reply:
okay leaving placeholder for now. with default value DD/MM/YYYY.